### PR TITLE
style(binding-netconf): fix eslint issues

### PR DESCRIPTION
**Pull request by JKRhb**
This PR fixes a number of eslint issues in the `binding-netconf` package.

### CODE REVIEW

File: packages/binding-netconf/src/xpath2json.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR W3C-20150513
  ********************************************************************************/
-var util = require("util");
+const util = require("util");
```
Comment (relu91) [inline packages/binding-netconf/src/xpath2json.ts:15]: Can't we use `import` here?

**Review by relu91 — CHANGES_REQUESTED** [review]
Thanks!!  just one minor concern below:

File: packages/binding-netconf/src/xpath2json.ts (hunk 1 above)
Comment (JKRhb) [reply packages/binding-netconf/src/xpath2json.ts:15]: I noticed that the import can actually be removed as `util` isn't used anywhere :) I will push an updated commit soon.

Comment (relu91) [reply packages/binding-netconf/src/xpath2json.ts]: I covered the change :)

**Review by relu91 — APPROVED** [review]
Sorry, the PR got polluted with the latest changes on the master cause I branched from it to cover my previous review. I the pr now looks ok.

**Comment by JKRhb** [comment]
Oh, no problem! Should I perform I rebase before this is going to be merged?

**Comment by relu91** [comment]
> Oh, no problem! Should I perform a rebase before this is going to be merged?

Why not 😃 👍🏻

**Comment by JKRhb** [comment]
Done :)

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**